packages/core-ethereum/src/commitment.ts

```diff
@@ -4,8 +4,8 @@
 //
 // We need to persist this string of commitments in the database, and support
 // syncing back and forth with those that have been persisted on chain.
-import { debug, Hash, HoprDB, iterateHash, recoverIteratedHash, toU8a, u8aConcat, UINT256 } from '@hoprnet/hopr-utils'
-import { deriveCommitmentSeed } from '@hoprnet/hopr-utils'
+import { debug, Hash, HoprDB, toU8a, u8aConcat, U256, recoverIteratedHash, iterateHash } from '@hoprnet/hopr-utils'
+import { derive_commitment_seed } from '@hoprnet/hopr-core/lib/core_crypto.js'
```

Comment:
This is considered unsafe. Please export `derive_commitment` in `core/index.ts` and import with

```
import { derive_commitment_seed } from '@hoprnet/hopr-core'